src/ontology/mondo-ingest.Makefile

```diff
@@ -275,3 +274,32 @@ mappings/mondo-sources-all-lexical.sssom.tsv: $(SCRIPTSDIR)/match-mondo-sources-
 	# The $^ includes the python script, basically all paramters after the colon :
 
 lexical_matches: mappings/mondo-sources-all-lexical.sssom.tsv
+
+#############################
+###### Slurp pipeline #######
+#############################
+slurp/:
+	mkdir -p $@
+
+# TODO: Move this to Makefile. how, when I shouldn't edit it from this repo?
+.PHONY: component-download-mondo.owl
+component-download-mondo.owl: | $(TMPDIR)
+	if [ $(MIR) = true ] && [ $(COMP) = true ]; then $(ROBOT) merge -I http://purl.obolibrary.org/obo/mondo.owl \
+	annotate --ontology-iri $(ONTBASE)/$@ $(ANNOTATE_ONTOLOGY_VERSION) -o $(TMPDIR)/$@.owl; fi
+
+# TODO: Move this to Makefile. how, when I shouldn't edit it from this repo?
+$(COMPONENTSDIR)/mondo.owl: component-download-mondo.owl
+	if [ $(COMP) = true ] ; then if cmp -s $(TMPDIR)/component-download-mondo.owl.owl $@ ; then echo "Component identical."; else echo "Component is different, updating." && cp $(TMPDIR)/component-download-mondo.owl.owl $@; fi; fi
+
+# min-id: the next available Mondo ID
+slurp/%.tsv: $(COMPONENTSDIR)/%.owl $(TMPDIR)/mondo.sssom.tsv $(REPORTDIR)/mirror_signature-mondo.tsv | slurp/
+	python $(SCRIPTSDIR)/migrate.py \
+	--ontology-path $(COMPONENTSDIR)/%.owl \
+	--sssom-map-path $(TMPDIR)/mondo.sssom.tsv \
+	--min-id 123000 \
```

Comment:
@matentzn How did you ascertain that `123000` was the next `min_id`. I imagine that this will be outdated soon. Won't we want some way to programmatically do this? How best to do that? 

**Possible solutions**
(a) Have the Python script figure it out, (b) have some other `make` goal that creates a file with this number in it, (c) have `update_repo` add a variable to `Makefile` for this?, (d) Update some `make` goal to insert this number into some type of Mondo config `.yml` somewhere?

I'm leaning towards (d).

Reply:
No no, this is not the _next_ available. This is the _minimum_ id. So basically, your script must take the minimum ID and increment the number until an ID is found that is not already taking but the ontology (mirror-signature)

Reply:
(this is just so that no ID smaller than `min` is chosen, even if it is free.

Reply:
Ah, ok. I misunderstood. So this is not likely to change. And if it does change, would not be very often.